Issue/mdot 256

David-Strupinski opened the pull request.

abztrakt commented:
Let's do a refactor of the regex after lookin at this: https://wiki.cac.washington.edu/display/SMW/UW+NetID+name+space+update+proposal+-+draft . Also, on the form, since the error message appears below the fold of the browser, we might need a notification at top that indicates there are fields that need to be corrected.

abztrakt commented:
Also, you somehow got a duplicate commit in this, "adds alterate Dockfile for M1 devices" is already on develop as commit 9bc4cc930de880c1e4a6a77058d6d18c353f5640

David-Strupinski commented:
I added the notification on the top of the form for any errors that occur below the fold and changed the regex to old accept personal and shared netIDs. Couldn't find a good way to remove the duplicate commit so far.
